README.md

```diff
@@ -90,6 +90,7 @@ The topics will be inferred by the browser. The browser will leverage a classifi
         * e.g., site A might see topic ‘cats’ for the user, but site B might see topic ‘automobiles’. It’s difficult for the two to determine that they’re looking at the same user.
     * The beginning of a week is per-user, per-site, and per-epoch. That is, for the same user, site A may see the new week's topics introduced at a different time than site B, and for the same user and site, the duration of a topic may not be exactly one week. This is to make it harder to correlate the same user across sites via the time that they change topics, or via the time interval between two changes.
 * Not every API caller will receive a topic. Only callers that observed the user visit a site about the topic in question within the past three weeks can receive the topic. If the caller (specifically the site of the calling context) did not call the API in the past for that user on a site about that topic, then the topic will not be included in the array returned by the API. The exception to this filtering is the 5% random topic, that topic will not be filtered.
+    * Each epoch is automatically deleted within four weeks. The exact deletion time is per-user, per-site, and per-epoch. Active users won't notice this deletion because only the most recent three epochs are exposed, covering just three weeks of data. But for inactive user, this ensures that the specific time an epoch is deleted can't be used to track or identify them across different websites.
```

Comment:
Suggest removing text after the second sentence. I think it's a bit confusing and doesn't really add much to the explainer.